src/layer/Popup.js

```diff
@@ -286,7 +277,7 @@ L.Popup = L.Layer.extend({
 		var prefix = 'leaflet-popup',
 		    container = this._container = L.DomUtil.create('div',
 			prefix + ' ' + (this.options.className || '') +
-			' leaflet-zoom-' + (this._zoomAnimated ? 'animated' : 'hide'));

```

Comment:
I think we want to keep that.
Even if the Popup itself does not have a `zoomAnimation` option, we want to listen to the animation capabilities of the browser and hide the popup when no animation will be made.